docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -26,18 +26,18 @@ The following table lists the available GX Cloud Expectations.
 | Data Integrity     | `expect_column_pair_values_to_be_equal`                   | Expect the values in column A to be the same as column B.                                                                              |
 | Data Integrity     | `expect_multicolumn_sum_to_equal`                         | Expect that the sum of row values in a specified column list is the same for each row, and equal to a specified sum total.             |
 | Distribution       | `expect_column_pair_values_A_to_be_greater_than_B`        | Expect the values in column A to be greater than column B.                                                                             |
-| Distribution       | `expect_column_values_to_be_between`                      | Expect the column entries to be between a minimum value and a maximum value (inclusive).                                               |
+| Distribution       | `expect_column_values_to_be_between`                      | Expect the column entries to be between a minimum value and a maximum value.                                               |
```

Comment:
**note for reviewers**: I removed "(inclusive)" because the user can make these expectations either inclusive or exclusive depending on how they set "Strict Min" / "Strict Max" in the UI